proxy/ProxyClientSession.h

```diff
@@ -119,6 +121,12 @@ class ProxyClientSession : public VConnection
     return m_active;
   }
 
+  bool
+  is_draining() const
+  {
+    return http_client_session_draining;
+  }
```

Comment:
I don't think there's any value in disguising the global variable here. Just use it directly where you need it.